corehq/apps/analytics/urls.py

```diff
@@ -0,0 +1,5 @@
+from django.conf.urls import *
+
+urlpatterns = patterns('corehq.apps.analytics.views',
+    (r'^hubspot/click-deploy/$', 'hubspot_click_deploy'),
```

Comment:
minor but we usually add a `name` field https://docs.djangoproject.com/en/1.8/topics/http/urls/#naming-url-patterns using the `url`